docs/getting_started.rst

```diff
@@ -1,8 +1,11 @@
+.. _getting_started:
+
 Getting Started
 ===============
 
-If you have not already done so, create/activate a `virtualenv`_. Unless otherwise stated, assume all terminal code
-below is executed within the virtualenv.
+If you have not already done so, create/activate a `virtualenv`_. Unless
+otherwise stated, assume all terminal code below is executed within the
+virtualenv. The virtualenv will need Python 3.6+ as it uses dbt under the hood.
```

Comment:
^ I am all for using newer versions of python, but I saw that 3.5 was referenced elsewhere in this repo. Also, we run warehousetransform jobs (w/ dbt) using Python3.7 on Jenkins. Does it make sense to standardize? I understand this might not be the best time/place for this.

Reply:
Hmm, I thought I'd removed all references to 3.5, but it's what the new cookiecutter projects get build as by default. I'll take a look and remove it if it's still there. 

I don't see any reason why we shouldn't suggest the lowest working version, though. I'm happy to test 3.6 - 3.8 if they all work now.